lib/Guideline/index.js

```diff
@@ -44,8 +45,14 @@ function Guideline({ children, title, onToggle, className, dir }) {
     'mr-auto': dir === 'rtl ml-0'
   });
 
+  const animatedStyle = useSpring({
+    opacity: visible ? 1 : 0,
+    from: { opacity: 0 },
+    config: { duration: 300 }
+  });
+
   return (
-    <div className={classNames} dir={dir}>
+    <animated.div style={animatedStyle} className={classNames} dir={dir}>
```

Comment:
💚 I'd spread a style prop here too, might not ever be used, but if we need it its there!